tests/cleanup/remove_old_objects_alibaba.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+# Script removes ECS Instances, Custom Images and OSS files older than
+# HOURS_LIMIT (24 hours by default) from Alibaba cloud
+#
+. /usr/share/beakerlib/beakerlib.sh
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        ALI_DIR=`mktemp -d /tmp/alicloud.XXXXX`
+
+        if [ -z "$ALICLOUD_ACCESS_KEY" ]; then
+            rlFail "ALICLOUD_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_ACCESS_KEY is configured"
+        fi
+
+        if [ -z "$ALICLOUD_SECRET_KEY" ]; then
+            rlFail "ALICLOUD_SECRET_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_SECRET_KEY is configured"
+        fi
+
+        ALICLOUD_BUCKET="${ALICLOUD_BUCKET:-composer-test}"
+        rlLogInfo "ALICLOUD_BUCKET=$ALICLOUD_BUCKET"
+
+        # VMs older than HOURS_LIMIT will be deleted
+        HOURS_LIMIT="${HOURS_LIMIT:-24}"
+        TIMESTAMP=`date -u -d "$HOURS_LIMIT hours ago" '+%FT%T'`
+        rlLogInfo "HOURS_LIMIT=$HOURS_LIMIT"
+        rlLogInfo "TIMESTAMP=$TIMESTAMP"
+
+        for package in jq; do
+            if ! rlCheckRpm "$package"; then
+                rlRun -t -c "dnf -y install $package"
+                rlAssertRpm "$package"
+            fi
+        done
+
+        # use the CLI b/c Ansible modules are not yet upstream and are unreliable
+        TAR_FILE="aliyun-cli-linux-3.0.32-amd64.tgz"
+        curl -L https://github.com/aliyun/aliyun-cli/releases/download/v3.0.32/$TAR_FILE > $ALI_DIR/$TAR_FILE
```

Comment:
:-1: on that idea. 

- using the latest version is prone to breaking the script when the cli decides to implement some of these commands differently. Not very likely but still possible
- the tar file names are not consistent between versions. There are slight differences in suffix of these files between older versions (since when this PR was originally opened) and the current ones. Seeing this history of release changes makes me pretty confident that would change again sometime in the future. 
- using GitHub API is overkill IMO. The right course of action is getting the ansible modules supported upstream so we don't have to rely on the cli. 

With a version lock the only thing that could go wrong is someone deleting the tarball from GitHub. We can always host this somewhere else (e.g. in our own S3) however that doesn't guarantee that the files won't become missing over time. We've seen breaking changes in our infra in the past (incl changing access scope).